Cargo.toml

```diff
@@ -37,7 +37,7 @@ once_cell = "1.17"
 [dev-dependencies]
 anyhow = "1"
 assert_cmd = "2.0"
-sp-storage = "11.0.0"
+sp-storage = { git = "https://github.com/paritytech/substrate" }
```

Comment:
nit: Would the latest release `13.0.0` work here?

Reply:
yeah but I changed it because it doesn't make sense to make the dependency tree even bigger :)

Reply:
I need to release a bunch crates for that and my goal is to use `master branches` here and check out crates.io version on each release.

Because it's so annoying to patch crates if we need some specific from substrate that isn't released.